app/views/admin/lead_provider_users/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Users for <%= @lead_provider.name %></h1>
+<table class="govuk-table">
+  <thead>
+  <tr class="govuk-table__row">
+    <th scope="col" class="govuk-table__header">First name</th>
+    <th scope="col" class="govuk-table__header">Last name</th>
+    <th scope="col" class="govuk-table__header">Email</th>
+    <th scope="col" class="govuk-table__header"></th>
+  </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+<% @users.each do |user| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= user.first_name %></td>
+      <td class="govuk-table__cell"><%= user.last_name %></td>
+      <td class="govuk-table__cell"><%= user.email %></td>
+      <td class="govuk-table__cell"><%= link_to "edit", edit_admin_lead_provider_user_path(@lead_provider, user) %></td>
+    </tr>
+  <% end %>
+  </tbody>
```

Comment:
Something is not quite right with whitespace here, in particular `tr` inside `thead` is not indented, and `users.each` tag is off.

Reply:
By default, rubymine does not intent children of ```html
body
thead
tbody
tfoot```. What is your preference?

Reply:
I strongly prefer we indent them. I find indentation really helpful when reading html, especially something as potentially complicated as tables.

Reply:
I guess you already found https://youtrack.jetbrains.com/issue/WEB-2230, but it seems changing it is not too difficult, although it needs to be done for each rubymine user :/

Reply:
Yeah I've found the settings. I'll see if I can track the editor config file?

Reply:
Sounds good to me :)